addons/website_sale/static/src/js/cart.js

```diff
@@ -61,12 +61,15 @@ publicWidget.registry.websiteSaleCart = publicWidget.Widget.extend({
      * @param {Event} ev
      */
     _onClickSuggestedProduct: function (ev) {
+        const dataset = ev.currentTarget.dataset;
+
         this.call('cart', 'add', {
-            productTemplateId: parseInt(ev.currentTarget.dataset.productTemplateId, 10),
-            productId: parseInt(ev.currentTarget.dataset.productId, 10),
-            isCombo: ev.currentTarget.dataset.productType === 'combo',
+            productTemplateId: parseInt(dataset.productTemplateId, 10),
+            productId: parseInt(dataset.productId, 10),
+            isCombo: dataset.productType === 'combo',
         }, {
             isBuyNow: true,
+            showQuantity: Boolean(dataset.showQuantity),
```

Comment:
This won't work unfortunately as `Boolean(<any string>)` is always `true` in javascript.
![Screenshot 2025-05-21 at 18 29 49](https://github.com/user-attachments/assets/6b1ed547-f64a-438b-ad4d-286f99d80a1f)
```suggestion
            showQuantity: dataset.showQuantity === 'True',  # Note the T uppercase, because python returns it with an uppercase
```